src/App.js

```diff
@@ -38,9 +41,60 @@ class App extends Component {
     }
   };
 
+  componentDidUpdate() {
+    if (this.state.user && !this.state.user.ratings) {
+      const id = this.state.user.id;
+      const url = "https://rancid-tomatillos.herokuapp.com/api/v2/users/";
+      this.fetchUserRatings(id, url);
+    }
+  }
+
+  fetchUserRatings(id, url) {
+    if (!url.includes('ratings')) {
+      url = `${url}${id}/ratings`;
+    }
+    fetch(url)
+      .then(response => response.json())
+      .then(data => data.ratings.map(rating => {
+        return {
+          id: rating.id,
+          userId: rating["user_id"],
+          movieId: rating["movie_id"],
+          rating: rating.rating,
+          createdAt: rating["created_at"],
+          updatedAt: rating["updated_at"]
+        }
+      }))
+      .then(ratings => {
+        this.state.user.ratings = ratings;
+        this.setState({user: this.state.user, ratings})
+        }
+      )
+      .catch(error => console.log(error))
+  }
+
+  postUserRating(id, url, rating) {
```

Comment:
Awesome , we are just flying through these iterations great work. Excited to get it running on my end.